_posts/2019-12-18-tfhub-0.7.0/tfhub-0.7.0.Rmd

```diff
@@ -0,0 +1,137 @@
+---
+title: "tfhub: R interface to TensorFlow Hub"
+description: > 
+ TensorFlow Hub is a library for the publication, discovery, and consumption of reusable parts of machine learning models. A module is a self-contained piece of a TensorFlow graph, along with its weights and assets, that can be reused across different tasks in a process known as transfer learning. 
+ 
+author:
+  - name: Daniel Falbel
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: tfhub
+date: 12-18-2019
+categories:
+  - Keras
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/tfhub.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+We are pleased to announce that the first version of [tfhub](https://github.com/rstudio/tfhub) is now on CRAN. tfhub is an R interface to TensorFlow Hub -  library for the publication, discovery, and consumption of reusable parts of machine learning models. A module is a self-contained piece of a TensorFlow graph, along with its weights and assets, that can be reused across different tasks in a process known as transfer learning. 
+
+The CRAN version of tfhub can be installed with:
+
+```{r}
+install.packages("tfhub")
+```
+
+After installing the R package you need to install the TensorFlow Hub python package. You can do it by running:
+
+```{r}
+tfhub::install_tfhub()
+```
+
+> **Note**: If you are using the most recent CRAN release of reticulate (1.14.0), tfhub will be automatically installed when you `library` it for the first time.
```

Comment:
I would remove this comment, most people reading this are not going to have the latest `reticulate` version yet.

Once we bump the min-reticulate required version, we can change this post to remove `install_tfhub()`.